README.md

```diff
@@ -1,70 +1,66 @@
-# Getting Started with Create React App
+# React-Capstone
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> Fetch Covid API,
+> Map all data from API into ino components
+> Rendered out routes based on id
+> Display COVID cases about particular countries
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+-React-js
+-Redux
 
-### `npm start`
+## Online live link
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[Video to my demonstration](https://drive.google.com/file/d/1kzy7Sief6KqaPxV7TzpF9YeZRSjuab61/view?usp=sharing)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Getting Started
 
-### `npm test`
+To get a local copy up and running follow these simple example steps.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Setup
 
-### `npm run build`
+To get a local copy up and running follow these simple example steps.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+- Clone the repository to your computer using git clone [Git Clone Url](https://github.com/mrparkersson/react-capstone)
+- Cd into the folder
+- Run `npm install` to install the project's dependencies;
+- Run `npm start` to open the website using webpack live server or
+  open dist/index.html with live server.
+- Or you can Fork it instead, and create a pull request to suggest some changes
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Visit And Open Files
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[Visit Repo](https://github.com/mrparkersson/react-capstone)
 
-### `npm run eject`
+## Deployed Web site
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+[visit live link](covidapireactapp.netlify.app)
```

Comment:
- [ ] The link you added leads to a 404 page. This is because the `.md` file does not recognize this as a link. It sees it more as a file in your directory (see image).

  ![parker-404](https://user-images.githubusercontent.com/36542986/157611731-5a56241f-1aac-4cde-9fd3-9730f4a722d7.jpg)

  ## SUGGESTION
  Simply add `https://` to your link like so: `[visit live link](https://covidapireactapp.netlify.app)`